components/scream/src/share/property_checks/field_upper_bound_check.hpp

```diff
@@ -1,28 +1,30 @@
 #ifndef SCREAM_FIELD_UPPER_BOUND_CHECK_HPP
 #define SCREAM_FIELD_UPPER_BOUND_CHECK_HPP
 
-#include "share/field/field_property_checks/field_within_interval_check.hpp"
+#include "share/property_checks/field_within_interval_check.hpp"
 
 #include <limits>
 
 namespace scream
 {
 
 // Convenience implementation of check for interval (-\infty,U]. The class
-// inherits from FieldWithinIntervalCheck, and sets lower bound to "minus infinity"
-  
+// inherits from FieldWithinIntervalCheck, and sets lower bound to -infinity
 class FieldUpperBoundCheck: public FieldWithinIntervalCheck {
 public:
-  // No default constructor -- we need an upper bound.
-  FieldUpperBoundCheck () = delete;
-
   // Constructor with upper bound. By default, this property check
   // can repair fields that fail the check by overwriting nonpositive values
   // with the given upper bound. If can_repair is false, the check cannot
   // apply repairs to the field.
-  explicit FieldUpperBoundCheck (const double upper_bound,
-                                 bool can_repair = true) : 
-    FieldWithinIntervalCheck(-std::numeric_limits<double>::max(), upper_bound, can_repair)
+  FieldUpperBoundCheck (const Field& field,
+                        const double upper_bound,
+                        const bool can_repair = true,
```

Comment:
If you decide to describe allow_failures in the comments for the lower bound check constructor, do it here too.